src/components/shared/CallNodeContextMenu.js

```diff
@@ -364,7 +364,9 @@ class CallNodeContextMenu extends PureComponent<Props, State> {
     } = this.props;
 
     if (selectedCallNodeIndex === null) {
-      return <div />;
+      throw new Error(
```

Comment:
Instead of throwing an error, would you please change this to a `console.error` with the same message, and still return a `<div />`. We have stopped throwing errors in render functions because of the potential end-user impact, although it would be nice to be informed if we are using it incorrectly.